### DISCUSSION TITLE
Welcome to vscode-cy-helper Discussions!

### DISCUSSION THREAD

**Discussion by Shelex** [90poe/vscode-cy-helper#38] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by elciodalosto** [comment]
Hi!
Are you guys thinking of implementing the ".skip" option by clicking on the top of scenarios descriptions? 
The ".only" option that already exists is very useful.
![skip](https://user-images.githubusercontent.com/15917796/204001112-c6628d60-59ab-44b1-9487-9d81d6cde069.png)

**Reply by Shelex** [reply]
Hi @elciodalosto 

Due to publishing token expiration and missing option to regenerate new one I have forked plugin under my personal account:

[Cypress Helper (latest)](https://marketplace.visualstudio.com/items?itemName=shevtsov.vscode-cy-helper)
[Repository](https://github.com/Shelex/vscode-cy-helper)
In case you have found this extension via pack, it is also republished - [extension pack](https://marketplace.visualstudio.com/items?itemName=shevtsov.qa-extension-pack)

This feature is already available in forked version and could be enabled by setting `cypressHelper.menuItems.ItSkip` as `true`
